fms/utils/generation.py

```diff
@@ -245,3 +243,15 @@ def truncate_after_eos(
         index = eos_index[0]
         result = result[: index + 1]
     return result
+
+
+def trim_prefix(result: torch.Tensor) -> torch.Tensor:
```

Comment:
should have a unit test.

also we should probably at least have an optional pad_id arg. we've hard-coded zero in a few places elsewhere, but I don't think that's ideal

Reply:
done